sdk/storage/azure-storage-blob/azure/storage/blob/aio/download_async.py

```diff
@@ -157,14 +179,16 @@ class StorageStreamDownloader(object):  # pylint: disable=too-many-instance-attr
     """
 
     def __init__(
-            self, service=None,
+            self, client=None,
```

Comment:
if you are fine with seperating download.py for file and blob, we can remove **client** from constructor and only keep **clients**. (Currenly client is used to perform download in blob_operations, clients is used to get page_blob client, and perform get_page_ranges operation.)

Reply:
Yeah I think we should be some refactoring here - having both `client` and `clients` seems odd.
Though if it's not public facing it can be refactored at a later date.

Reply:
Rather than have a public attribute called `clients` - could we possibly rename this to something like `generated_api` and make it private?
That way if we refactor or rename it later, it's already private and we can do what we like with it. I also feel the name `clients` is confusing.
```
def __init__(client=None, generated_api=None, ..., **kwargs):
    self.client = client
    self._generated_api = generated_api
```

Reply:
How about just keep clients and remove client?